interface/billing/search_payments.php

```diff
@@ -211,7 +211,7 @@
 <html>
 <head>
 
-    <?php Header::setupHeader(['jquery-ui', 'datetime-picker']); ?>
+    <?php Header::setupHeader(['jquery-ui', 'datetime-picker', 'font-awesome']); ?>
```

Comment:
remove font-awesome (is automatically included)

Reply:
Done